index.js

```diff
@@ -108,7 +108,8 @@ function RingPop(options) {
     this.requestProxy = new RequestProxy({
         ringpop: this,
         maxRetries: options.requestProxyMaxRetries,
-        retrySchedule: options.requestProxyRetrySchedule
+        retrySchedule: options.requestProxyRetrySchedule,
+        enforceConsistency: options.enforceConsistency === undefined ? true : options.enforceConsistency
```

Comment:
Put this condition in the constructor of the RequestProxy object instead of the instantiator. If RequestProxy is ever instantiated outside of this call-site, you'd be changing expected behavior.

aka, make this look like:

```
enforceConsistency: options.enforceConsistency
```